docs/source/index.rst

```diff
@@ -0,0 +1,51 @@
+======================
+Autometa documentation
+======================
+
+| **Autometa: automated extraction of microbial genomes from individual shotgun metagenomes**
+| An automated binning pipeline for single metagenomes, in particular host-associated and highly complex ones. 
+
+| If you find Autometa useful to your work, please cite our Autometa_ paper:
+| Miller, I. J.; Rees, E. R.; Ross, J.; Miller, I.; Baxa, J.; Lopera, J.; Kerby, R. L.; Rey, F. E.; Kwan, J. C. Autometa: Automated extraction of microbial  genomes from individual shotgun metagenomes. *Nucleic Acids Research*, **2019**.  DOI: https://doi.org/10.1093/nar/gkz148
+
+Quickstart
+==========
+
+Run with below code::
+
+    python autometa.py metagenome.config
+
+Guide
+=====
+
+.. toctree::
+   :maxdepth: 1
+
+   install
+   databases
+   running-autometa
+   examining-results
+   Scripts/scripts
+   contribution-guidelines
+   api
+   license
+
+
+Contact
+-------
+For suggestions, queries or appreciation feel free to contact `Dr.JasonKwan <https://apps.pharmacy.wisc.edu/sopdir/jason_kwan/>`_
+at jason.kwan@wisc.edu
+
+Copyright and License
```

Comment:
You could probably just remove this section Since License is in the sidebar. At least remove the Copyright portion